src/main.rs

```diff
@@ -50,6 +51,17 @@ enum Opt {
         )]
         sort: SortOrder,
 
+        /// Analyze .ll files that were produced by eg. `RUSTFLAGS="--emit=llvm-ir" ./x.py build --stage 0 compiler/rustc`.
+        /// Supports globs, eg `./build/x86_64-unknown-linux-gnu/stage0-rustc/x86_64-unknown-linux-gnu/debug/deps/*.ll`
```

Comment:
This is probably a little more complicated of an example than you need to give in public docs. It's better suited for rustc-dev-guide IMO.

```suggestion
        /// Analyze .ll files produced by `RUSTFLAGS="--emit=llvm-ir"`.
        ///
        /// Supports globs, e.g. `./build/debug/deps/*.ll`
```

Reply:
Changed in #33.